account_analytic_sequence/README.rst

```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Can you adapt the readme file according the template https://github.com/OCA/maintainer-tools/tree/master/template/module

Otherwise :+1: